cpp/mrc/include/mrc/modules/properties/persistent.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * SPDX-FileCopyrightText: Copyright (c) 2021-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+namespace mrc::modules {
+class PersistentModule
```

Comment:
Should this be removed?

Reply:
No, its used to test if a module needs to be persisted by the Builder. Usage is in `builder.cpp::init_module`, I'd looked at making it a variable on the module, but at the end of the day, its going to be something known at compile time and shouldn't be mutable.

Reply:
** Add comment indicating its intentionally empty.